content/docs/04-clusters/04-palette-virtual-clusters.md

```diff
@@ -23,7 +23,7 @@ Palette provisions and orchestrates Palette Virtual Clusters and makes it easy t
 
 Palette also supports Day 2 operations such as upgrades, backup/restore and more, to keep Palette Virtual Clusters secure, compliant, and up to date. Additionally, it provides visibility into the workloads running inside your Palette Virtual Clusters and its associated costs.
 
-To get started and create your Palette Virtual Cluster, see the [Palette Virtual Cluster Quick Start](/clusters/palette-virtual-clusters/virtual-cluster-quickstart) page.
```

Comment:
@ritawatson, if you are changing a file name, then that requires a redirect. If we can avoid changing a file name, then we should.

Reply:
@karl-cardenas-coding I didn't know that. Could you help me create a redirect?

Reply:
Resolved